simpleflow/execute.py

```diff
@@ -8,6 +9,8 @@
 
 import psutil
 
+MAX_ARGUMENTS_JSON_LENGTH = 65536
```

Comment:
Is that value related to ARG_MAX http://porkmail.org/era/unix/arg-max.html ?
On my computer:
```
$ getconf ARG_MAX
2097152
```

If so, shouldn't this value ideally be more dynamic?

Reply:
No... There's a hard limit much lower than this :roll_eyes: